queue_job/models/queue_job.py

```diff
@@ -116,6 +121,14 @@ def _compute_record_ids(self):
         for record in self:
             record.record_ids = record.records.ids
 
+    @api.depends("date_done")
+    def _compute_exec_time(self):
+        for record in self:
+            if not record.date_started or not record.date_done:
+                record.exec_time = False
+                continue
+            record.exec_time = (record.date_done - record.date_started).total_seconds()
```

Comment:
I'd prefer not to use a computed field (see https://github.com/OCA/queue/pull/305).
It should be written by https://github.com/OCA/queue/blob/563db616ff33496b75e82b98f9734c8c66637ce4/queue_job/job.py#L685-L690

Reply:
ahhh ok, you are right! Then just this would do the work:
```py
self.exec_time = (self.date_done - self.date_started).total_seconds()
```

Reply:
done